docs/_docs/index.md

```diff
@@ -10,36 +10,53 @@ Jekyll is a static site generator. It takes text written in your
 favorite markup language and uses layouts to create a static website. You can
 tweak the site's look and feel, URLs, the data displayed on the page, and more. 
 
-## Prerequisites
+## Setup
 
-Jekyll requires the following:
-
-* Ruby version **{{ site.data.ruby.min_version }}** or higher
-* RubyGems
-* GCC and Make
-
-See [Requirements]({{ '/docs/installation/#requirements' | relative_url }}) for guides and details.
-
-## Instructions
-
-1. Install all [prerequisites]({{ '/docs/installation/' | relative_url }}).
+1. Install all [prerequisites]({{ '/docs/installation/' | relative_url }}) (basicly Ruby **{{ site.data.ruby.min_version }}** or higher, GCC and Make.
 2. Install the jekyll and bundler [gems]({{ '/docs/ruby-101/#gems' | relative_url }}).
 ```sh
 gem install jekyll bundler
 ```
-3. Create a new Jekyll site at `./myblog`.
+
+## Create a site
+
+1. Create a new Jekyll site at `./my-awesome-site`.
 ```sh
-jekyll new myblog
+jekyll new my-awesome-site
```

Comment:
I don't see the need to change `myblog` to something else.
Though the only issue I have with `my-awesome-site` is that it's a *longer* name..